dev_guide/templates.adoc

```diff
@@ -444,8 +444,11 @@ apiVersion: "v1"
 ...
 labels:
   template: "cakephp-mysql-example" <1>
+  app: "${NAME}" <2>
```

Comment:
assuming you agree we should support key substitution, update this to illustrate that too and then lgtm.

Reply:
done